src/controllers/auth.js

```diff
@@ -259,9 +260,61 @@ const obtainAccessToken = async (req, res) => {
   }
 };
 
+const refreshAccessToken = async (req, res) => {
+  try {
+    // extract the payload from the token and verify it
+    const payload = await decodeToken(
```

Comment:
What if decoding the token fails?

Reply:
Then the `catch` section will get the error thrown by the `decodeToken` function and it will send the error to the cleint 

```
res.status(401).send({ message: 'Invalid refresh token' });
```